.gitignore

```diff
@@ -16,7 +16,8 @@ env/
 venv/
 ENV/
 VENV/
-
+hnn_core/mod/*
```

Comment:
I think this folder is already in the ignore list. Is there a reason to include it again?